util/cmdtest/cmds.go

```diff
@@ -35,147 +42,118 @@ func init() {
+func (suite *IntegrationSuite) StartServe(args ...string) (io.Reader, io.WriteCloser) {
+	require := suite.Require()
+
 	r, outputWriter := io.Pipe()
 	buf := &bytes.Buffer{}
 	tee := io.TeeReader(r, buf)
 
 	args = append([]string{"serve"}, args...)
 
-	cmd := exec.CommandContext(ctx, lookoutBin, args...)
+	cmd := exec.CommandContext(suite.Ctx, lookoutBin, args...)
 	cmd.Stdout = outputWriter
 	cmd.Stderr = outputWriter
 
 	w, err := cmd.StdinPipe()
-	if err != nil {
-		fmt.Println("can't start server:")
-		fmt.Println(err)
-		os.Exit(1)
-	}
+	require.NoError(err, "can't start server")
 
 	err = cmd.Start()
-	if err != nil {
-		ioutil.ReadAll(tee)
-		fmt.Println("can't start server:")
-		fmt.Println(err)
-		fmt.Printf("output:\n %s", buf.String())
-		os.Exit(1)
-	} else {
-		go func() {
-			if err := cmd.Wait(); err != nil {
-				// don't print error if analyzer was killed by cancel
-				if ctx.Err() != context.Canceled {
-					ioutil.ReadAll(tee)
-					fmt.Println("server exited with error:", err)
-					fmt.Printf("output:\n%s", buf.String())
-					failExit()
-				}
+	require.NoError(err, "can't start server")
+
+	go func() {
+		if err := cmd.Wait(); err != nil {
+			// don't print error if analyzer was killed by cancel
+			if suite.Ctx.Err() != context.Canceled {
+				fmt.Println("server exited with error:", err)
+				fmt.Printf("output:\n%s", buf.String())
+				// T.Fail cannot be called from a goroutine
+				failExit()
 			}
-		}()
-	}
+		}
+	}()
 
 	return tee, w
 }
 
 // RunCli runs lookout subcommand (not a server)
-func RunCli(ctx context.Context, cmd string, args ...string) io.Reader {
+func (suite *IntegrationSuite) RunCli(cmd string, args ...string) io.Reader {
 	args = append([]string{cmd}, args...)
 
 	var out bytes.Buffer
-	reviewCmd := exec.CommandContext(ctx, lookoutBin, args...)
-	reviewCmd.Stdout = &out
-	reviewCmd.Stderr = &out
-
-	err := reviewCmd.Run()
-	if err != nil {
-		fmt.Println("review command returned error")
-		fmt.Println(err)
-		fmt.Printf("output:\n %s", out.String())
-		failExit()
-	}
+	cliCmd := exec.CommandContext(suite.Ctx, lookoutBin, args...)
+	cliCmd.Stdout = &out
+	cliCmd.Stderr = &out
+
+	err := cliCmd.Run()
+	suite.Require().NoErrorf(err,
+		"'lookout %s' command returned error. output:\n%s",
+		strings.Join(args, " "), out.String())
 
 	return &out
 }
 
 // ResetDB recreates database for the test
-func ResetDB() {
+func (suite *IntegrationSuite) ResetDB() {
+	require := suite.Require()
+
 	db, err := sql.Open("postgres", "postgres://postgres:postgres@localhost:5432/lookout?sslmode=disable")
-	if err != nil {
-		fmt.Println("can't connect to DB:", err)
-		os.Exit(1)
-	}
+	require.NoError(err, "can't connect to DB")
 
 	_, err = db.Exec("DROP SCHEMA public CASCADE;")
-	noDBErr(err)
+	require.NoError(err, "can't execute query")
 	_, err = db.Exec("CREATE SCHEMA public;")
-	noDBErr(err)
+	require.NoError(err, "can't execute query")
```

Comment:
Do you think making it something like "can't execute SQL: '%s'" would increase ability to debug the failure?